lib/datetimelib/datetimelib.c

```diff
@@ -0,0 +1,105 @@
+#include "datetimelib.h"
```

Comment:
It's ok to make it datetime

Reply:
renamed to datetime.h, build errors fixed (appeared after merge dev branch, caused by using old furi_hal_rtc datetime functions)